### PR TITLE
Run bugzilla automation script on github actions

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/6368
closes #6368

### CODE REVIEW

File: .github/settings.toml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[default.email]
+from = "no-reply@redhat.com"
+subject = "[Pulp CI] Bugzilla automation script"
```
Comment (rochacbruno) [inline .github/settings.toml]: Subject is not updated here :)

File: .github/send_email.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    #     click.echo(msg)
+    #     return
+
+    msg.attach(part2)  # html is included only if not --test
+    return server.sendmail(mail_from, mail_to, msg.as_string())
+
+
+if __name__ == '__main__':
+    send_html_email(
+        "dkliban@redhat.com, ttereshc@redhat.com, rchan@redhat.com",
```
Comment (rochacbruno) [inline .github/send_email.py]:
```suggestion
        ["dkliban@redhat.com", "ttereshc@redhat.com", "rchan@redhat.com"],
```

File: .github/send_email.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        "Pulp CI job failed",
```
Comment (rochacbruno) [inline .github/send_email.py]: This is the text based message, text only with `\n` allowed

File: .github/send_email.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        "Pulp CI job failed - https://github.com/pulp/pulp-ci/actions",
```
Comment (rochacbruno) [inline .github/send_email.py]: this allows html tags
```suggestion
        "<h1>Pulp CI job failed</h1> - https://github.com/pulp/pulp-ci/actions",
```

File: .github/settings.toml (hunk 1 above)
Comment (rochacbruno) [reply .github/settings.toml]: nm, just notice that this is the proper subject :)

**Comment by fabricio-aguiar** [comment]
Thank you @rochacbruno

File: .github/workflows/bugzilla_redmine.yml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: Bugzilla automation script
+        env:
+          REDMINE_BUGZILLA_CONF: ${{ secrets.REDMINE_BUGZILLA_CONF }}
```
Comment (fao89) [inline .github/workflows/bugzilla_redmine.yml:24]: Need secret!

Comment (daviddavis) [reply .github/workflows/bugzilla_redmine.yml:24]: What format does this need to be in?

Comment (fao89) [reply .github/workflows/bugzilla_redmine.yml:24]: INI format:
https://github.com/pulp/pulp-ci/blob/master/ci/redmine_bugzilla.py#L25-L38

Comment (daviddavis) [reply .github/workflows/bugzilla_redmine.yml:24]: The secret for the redmine key is `REDMINE_API_KEY` and it'll be `BUGZILLA_USERNAME` and `BUGZILLA_PASSWORD` for the bz credentials.

Comment (daviddavis) [reply .github/workflows/bugzilla_redmine.yml]: So I've set `BUGZILLA_API_KEY` as a secret on this repo. They're disabling use of username+password in favor of api keys. You should be able to do this:

```python
RHBugzilla(f"{BUGZILLA_URL}/xmlrpc.cgi", api_key=api_key)
```

**Pull request merged**